cirq/ops/swap_gates.py

```diff
@@ -248,10 +248,15 @@ def __repr__(self):
                                              self._global_shift)
 
 
-def ISwapRotation(angle_rads: value.TParamVal) -> ISwapPowGate:
+def riswap(rads: value.TParamVal) -> ISwapPowGate:
     """Returns gate with matrix exp(+i angle_rads (X⊗X + Y⊗Y) / 2)."""
-    pi = sympy.pi if protocols.is_parameterized(angle_rads) else np.pi
-    return ISwapPowGate()**(2 * angle_rads / pi)
+    pi = sympy.pi if protocols.is_parameterized(rads) else np.pi
+    return ISwapPowGate()**(2 * rads / pi)
+
+
+@deprecated(deadline='v0.7.0', fix='Use cirq.riswap, instead.')
```

Comment:
v0.7 is the version this deprecation will appear in for the first time. Make it v0.8

Reply:
Done.